src/config/core.clj

```diff
@@ -95,7 +89,8 @@
       (read-config-file "config.edn")
       (read-env-file ".lein-env")
       (read-env-file (io/resource ".boot-env"))
-      (read-env-file (:config env-props))
+      (when (:config env-props)
+        (read-env-file (:config env-props) true))
```

Comment:
The other files are optional, but if `(:config env-props)` is set, we want fail if the file doesn't exist. So I introduced an additional optional argument for `read-env-file` that prevents the condition `(.exists env-file)` but requires the file to be read successfully or throw an `FileNotFoundException`.

Reply:
sounds reasonable, might be worth documenting the new flag in the readme

Reply:
The README.md doesn't mention the internal `read-env-file` function.
If someone would have imported and used it, the behavior of calling it with a single argument is unchanged.

We could add a docstring to the function, but I think the function is so small and well readable without any doc

Reply:
ok yeah let's leave it as is then